src/toolchain/rust.rs

```diff
@@ -36,7 +36,8 @@ const DEFAULT_XTENSA_RUST_REPOSITORY: &str =
 /// Xtensa Rust Toolchain API URL
 const XTENSA_RUST_LATEST_API_URL: &str =
     "https://api.github.com/repos/esp-rs/rust-build/releases/latest";
-const XTENSA_RUST_API_URL: &str = "https://api.github.com/repos/esp-rs/rust-build/releases";
+const XTENSA_RUST_API_URL: &str =
+    "https://api.github.com/repos/esp-rs/rust-build/releases?page=1&per_page=1000";
```

Comment:
They do state 100 as the maximum, though I'm not sure it matters in practice. Probably they will just drop it to 100 on the server.